docs/content/preview/develop/multi-cloud/multicloud-setup.md

```diff
@@ -0,0 +1,215 @@
+
+## Topology
+
+For illustration, let us set up a 6-node universe across AWS-`us-west`, Google Cloud Provider-`us-central`, and Microsoft Azure- `us-east`.
+
+![Multi-cloud Yugabyte](/images/develop/multicloud/multicloud-topology.png)
+
+<!-- begin: nav tabs -->
+{{<nav/tabs list="local,cloud,anywhere" active="local"/>}}
+
+{{<nav/panels>}}
+{{<nav/panel name="local" active="true">}}
+<!-- BEGIN: local cluster setup instructions -->
+
+## Set up VPC peering
+
+{{<warning title="Remember">}}
+Although for the current example, you **do not** have to set up VPC peering, for different clouds to be able to talk to each other, you need to set up multi-cloud VPC peering through a VPN tunnel.
+{{</warning>}}
+
+YugabyteDB requires a particular [set of firewall ports](../../../yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes/#ports) to be opened for cluster operations.
+
+You should use non-overlapping [Classless Inter-Domain Routing (CIDR)](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#Further_reading) blocks for each subnet across different clouds.
+
+All public cloud providers enable VPN tunneling across VPCs and their subnets to enable network discovery. As an example, see [VPN between two clouds](https://medium.com/google-cloud/vpn-between-two-clouds-e2e3578be773).
+
+## Set up the multi-cloud universe
+
+We are going to set up a 6-node universe with 2 nodes each in AWS,GCP and Azure.
+
+### First cloud - AWS
+
+{{<setup/local
+    numnodes="2"
+    rf="3"
+    basedir="/tmp/ydb-aws-"
+    status="no"
+    dataplacement="no"
+    locations="aws.us-west-2.us-west-2a,aws.us-west-2.us-west-2b">}}
+
+### Second cloud - GCP
+
+{{<note title="Note">}} These nodes in GCP will join the cluster in AWS (127.0.0.1) {{</note>}}
+
+{{<setup/local
+    numnodes="2"
+    rf="3"
+    destroy="no"
+    ips="127.0.0.3,127.0.0.4"
+    masterip="127.0.0.1"
+    basedir="/tmp/ydb-gcp-"
+    dataplacement="no"
+    status="no"
+    locations="gcp.us-central-1.us-central-1a,gcp.us-central-1.us-central-1b">}}
+
+### Third cloud - Azure
+
+{{<note title="Note">}} These nodes in Azure will join the cluster in AWS (127.0.0.1) {{</note>}}
+
+{{<setup/local
+    numnodes="2"
+    rf="3"
+    destroy="no"
+    ips="127.0.0.5,127.0.0.6"
+    masterip="127.0.0.1"
+    basedir="/tmp/ydb-azu-"
+    dataplacement="yes"
+    status="yes"
+    locations="azu.us-east-1.us-east-1a,azu.us-east-1.us-east-1b">}}
+
+## Destroy the universe
+
+After exploring the multi-cloud setup locally, you can destroy the cluster using the following command
+
+```bash
+for d in {aws,gcp,azu}-{1,2} ; do ./bin/yugabyted destroy --base_dir=/tmp/ydb-${d} ; done
+```
+
+<!-- END: local cluster setup instructions -->
+{{</nav/panel>}}
+{{<nav/panel name="cloud">}} {{<setup/cloud>}} {{</nav/panel>}}
+{{<nav/panel name="anywhere">}}
+<!-- BEGIN: YBA cluster setup instructions -->
+
+## Set up instance VMs
```

Comment:
Why not simply link to YBA page rather than duplicate setup here

Reply:
Probably that might be a better idea. Will first get the local cluster working correctly and then move this out